modules/team_planner/app/views/team_planner/team_planner/upsale.html.erb

```diff
@@ -5,5 +5,6 @@
                feature_title: t('team_planner.upsale.title'),
                feature_description: t('team_planner.upsale.description'),
                feature_reference: 'team_planner',
-               feature_image: 'enterprise/team-planner.png'
+               feature_image: 'enterprise/team-planner.png',
```

Comment:
The image was only a placeholder and can be removed now.

Reply:
Hi @HDinger As I understood In Rails we should add videos to the public folder, https://apidock.com/rails/v5.2.3/ActionView/Helpers/AssetTagHelper/video_tag I have tested it by adding the video to assets folder, but it was not displayed on the page. Is there another way to show a video in this case?

Can we have a higher resolution image for other places? Should I contact product team for it?

Reply:
@bsatarnejad In the documentation it says "The sources can be full paths or files that exist in your public videos directory." For our case we need the full path option. On production, images and videos needs be processed by the asset pipeline, in order to display them correctly. As I said, the `public` folder is not included in that pipeline resulting in the video not being found.
I just tested it myself and it worked correctly. Be sure to add the video to `openproject/app/assets/videos` and **not** to the frontend folder. Further you need to restart your server once you did that.